src/components/AddServicePointModal/AddServicePointModal.js

```diff
@@ -23,8 +23,12 @@ class AddServicePointModal extends React.Component {
     intl: intlShape.isRequired,
   }
 
-  state = {
-    selection: {},
+  constructor(props) {
+    super(props);
```

Comment:
`react/state-in-constructor` was turned off in `eslint-config-stripes` 5.2.0. If you upgrade to that version, you can revert these changes.